deploy/charts/firefly/scripts/ff-db-migrations.sh

```diff
@@ -3,29 +3,34 @@
 # Install deps
 apk add postgresql-client curl jq
 
+echo "Provided connection string: '${PSQL_URL}'"
+
 # Extract the database name from the end of the PSQL URL, and check it's there
-DB_NAME=`echo ${PSQL_URL} | sed 's/^.*\///'`
-COLONS=`echo -n $DB_NAME | sed 's/[^:]//g'`
+DB_PARAMS=`echo ${PSQL_URL} | sed 's!^.*/!!'`
+DB_NAME=`echo ${DB_PARAMS} | sed 's!?.*!!'`
 echo "Database name: '${DB_NAME}'"
+USER_NAME=`echo ${PSQL_URL} | sed 's!^.*//!!' | sed 's!:.*$!!'`
+echo "Username: '${USER_NAME}'"
```

Comment:
Does this still work if the DB doesn't require auth? Know thats a bit unlikely / harder to test.

Reply:
Yeah that's a good point. Now, we need connection strings in at least this format:

"postgresql://[username]:[password]/[database]"

Looking at their doc (https://www.postgresql.org/docs/current/libpq-connect.html), supporting all formats of connection strings would be pretty difficult. Wondering what we need to support here? @peterbroadhurst and @hfuss  any insights from what you've previously deployed? It may also be easier to provide the PSQL connection info as separate chart values instead of winding up with all this `sed` `echo fun | sed 's/fun/suffering/'`.

Reply:
I think assuming the PSQL string will have basic auth creds is fine for now. Most users are going to have a setup like that when using the chart.